### ISSUE TITLE
[? Question] Identify the .pth for the best Hyperparam configuration

### ISSUE THREAD

**Issue by mmosc** [RUCAIBox/RecBole#1652]
Hi, and thank you for this great library!

Maybe I missed on this, but I would like to know the following:

## Describe the problem
When performing grid search with hyperopt, RecBole stores all model instances (i.e., a `.pth` file for all combinations of hyperparameters), which is great. However, How do I identify the model `.pth` file that achieved the best performance on the validation set **without looking at the console output**?

Cheers, 
Marta

**Comment by Sherry-XLL** [comment]
Hello, @mmosc! Thanks for your attention to RecBole!

We regret to tell you that RecBole cannot directly obtain the file path of the optimal parameters at present. We will improve this problem in future versions. Thank you again for your feedback and nice suggestions.

**Comment by mmosc** [comment]
Thanks for your quick reply! I see, that'd be a nice feature. 
Best, 
Marta
